README.md

```diff
@@ -13,10 +16,10 @@ NVTabular is designed to support Data Scientists and ML Engineers trying to trai
 It is also meant to help ML/Ops Engineers deploying models into production by providing:
 
 * Faster dataset transformation, allowing for production models to be trained more frequently and kept up to date, helping improve responsiveness and model performance.
-* Integration with model serving frameworks like [NVIDIA’s Triton Inference Server](https://github.com/NVIDIA/tensorrt-inference-server) to make model deployment easy.
-* Statistical monitoring of the dataset for distributional shift and outlier detection during production training or inference.
+* Integration with model serving frameworks like [NVIDIA’s Triton Inference Server](https://github.com/NVIDIA/tensorrt-inference-server) and [Tensorflow Serving](https://github.com/tensorflow/serving) to make model deployment easy. (WIP)
+* Statistical monitoring of the dataset for distributional shift and outlier detection during production training or inference. (WIP)
 
-The library is designed to be interoperable with both PyTorch and TensorFlow using batch data-loaders that we have developed as extensions of native framework code. NVTabular provides the option to shuffle data during preprocessing, allowing the data-loader to load large contiguous chunks from files rather than individual elements. This allows us to do per epoch shuffles orders of magnitude faster than a full shuffle of the dataset. We have benchmarked our data-loader at 100x the baseline item by item PyTorch dataloader and 3x the TensorFlow batch data-loader, with several optimizations yet to come in that stack.
+The library is designed to be interoperable with both PyTorch and TensorFlow using dataloaders that we have developed as extensions of native framework code. NVTabular provides the option to shuffle data during preprocessing, allowing the dataloader to load large contiguous chunks from files rather than individual elements. This allows us to do per epoch shuffles orders of magnitude faster than a full shuffle of the dataset.  Loading tabular data is almost always a training bottleneck and in our benchmarking we have seen 10x improvements in training time on GPU relative to the native dataloaders.  We also 
```

Comment:
The last sentence here seems cut off - maybe we should remove?
```suggestion
The library is designed to be interoperable with both PyTorch and TensorFlow using dataloaders that we have developed as extensions of native framework code. NVTabular provides the option to shuffle data during preprocessing, allowing the dataloader to load large contiguous chunks from files rather than individual elements. This allows us to do per epoch shuffles orders of magnitude faster than a full shuffle of the dataset.  Loading tabular data is almost always a training bottleneck and in our benchmarking we have seen 10x improvements in training time on GPU relative to the native dataloaders. 
```